### PR TITLE
LPS-151723 Move petra-encryptor to portal-encryptor

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina, Shuyang,

Re-submitted from: #679

Main Task: [LPS-147928](https://issues.liferay.com/browse/LPS-147928)
SubTask: [LPS-151723](https://issues.liferay.com/browse/LPS-151723)

Note: 
- this pull includes changes to liferay-release-tool-ee at Shuyang's repo here: 
https://github.com/shuyangzhou/liferay-release-tool-ee/pull/69

Thank you!

### CODE REVIEW

**Comment by julschong** [comment]
pushed with additional clean ups in project.xml, sf properties, and .classpath

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

File: git-commit/liferay-release-tool-ee [hunk 1]
```diff
@@ -1 +1 @@
-HEAD
\ No newline at end of file
+https://github.com/shuyangzhou/liferay-release-tool-ee/pull/69
```
Comment (julschong) [inline git-commit/liferay-release-tool-ee:1]: @tinatian is this correct?
Thank you!

**Comment by julschong** [comment]
pushed to open methods in Util

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
Could you please help us scan the test result?
Thank you!

**Comment by julschong** [comment]
rebased and resolved conflict

**Comment by catarineaguiar** [comment]
Hi @julschong no regressions were found.
 The formatting issue is:
 
 ```
[java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/package.json expected:<...	"@clayui/css": "3.5[4].0",
     [java] 		"@clayui/drop...> but was:<...	"@clayui/css": "3.5[3].0",
     [java] 		"@clayui/drop...>
     [java] 
     [java] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:435)
     [java] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:281)
```

**Comment by julschong** [comment]
@catarineaguiar 
Hi Catarine, 
I think that SF issue was caused by something else :) 
but someone else already fixed it it seems

I rebased and pushed again

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
not fixed yet :(

**Comment by julschong** [comment]
rebased again

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
Could you please help us scan the test result again just in case?
Thank you!

**Comment by catarineaguiar** [comment]
@julschong no regressions found.

**Comment by julschong** [comment]
@tinatian
Hi Tina, could you please help me review this pull?
Thank you!

File: source-formatter.properties [hunk 2]
```diff
@@ -382,6 +382,7 @@
         modules/apps/petra/petra-content,\
         modules/apps/petra/petra-doulos,\
         modules/apps/petra/petra-encryptor,\
+        modules/apps/petra/petra-http-invoker,\
```
Comment (tinatian) [inline source-formatter.properties:385]: @julschong The change is not consistent with your commit message.

File: modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyInfoLocalServiceTest.java [hunk 3]
```diff
@@ -75,7 +75,7 @@ public void testGetCompanyInfoKeyObj() {
 			_company.getCompanyId());
 
 		Assert.assertEquals(
-			Encryptor.deserializeKey(companyInfo.getKey()),
+			EncryptorUtil.deserializeKey(companyInfo.getKey()),
```
Comment (tinatian) [inline modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyInfoLocalServiceTest.java:78]: @julschong Please use "@Inject" here.

File: source-formatter.properties (hunk 2 above)
Comment (julschong) [reply source-formatter.properties]: @tinatian 
sorry for the mix up.

File: source-formatter.properties [hunk 4]
```diff
@@ -381,7 +381,6 @@
         modules/apps/petra/petra-apache-http-components,\
         modules/apps/petra/petra-content,\
         modules/apps/petra/petra-doulos,\
-        modules/apps/petra/petra-encryptor,\
```
Comment (julschong) [inline source-formatter.properties:384]: fixed here

File: modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyInfoLocalServiceTest.java [hunk 5]
```diff
@@ -114,6 +114,9 @@ public void testUpdateCompanyInfoKeyObj() {
 	@Inject
 	private static CounterLocalService _counterLocalService;
 
+	@Inject
```
Comment (julschong) [inline modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyInfoLocalServiceTest.java:117]: fixed here

File: modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyInfoLocalServiceTest.java [hunk 6]
```diff
@@ -114,6 +114,9 @@ public void testUpdateCompanyInfoKeyObj() {
...
+	private static Encryptor _encryptor;
+
```
Comment (julschong) [inline modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyInfoLocalServiceTest.java:119]: fixed here

File: modules/apps/company/company-test/build.gradle [hunk 7]
```diff
@@ -1,7 +1,6 @@
 dependencies {
 	testIntegrationCompile group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "default"
 	testIntegrationCompile project(":apps:layout:layout-api")
-	testIntegrationCompile project(":apps:petra:petra-encryptor")
```
Comment (julschong) [inline modules/apps/company/company-test/build.gradle:4]: removed dependency here

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by tinatian** [comment]
@catarineaguiar Please help me scan test results.

Thanks.
Tina

**Comment by catarineaguiar** [comment]
@julschong @tinatian no regressions found.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**